src/main/resources/org/jenkinsci/plugins/jx/pipelines/dsl/MavenFlow.groovy

```diff
@@ -211,10 +196,10 @@ class MavenFlow {
       dir = p.toString()
```

Comment:
FYI, `script.pwd()` is always going to return a `String`, so the above code is actually unneeded, but I decided not to remove it for now in case there was some reasoning I'm not clear on. =)

Reply:
@abayer thanks! I was getting funky errors when using this code; turns out the docker image had a cached old jenkins plugin that was making me doubt the groovy code ;)

Reply:
heheh. Good times.